docs/tutorials/13_quantum_random_access_optimizer.ipynb

```diff
@@ -0,0 +1,736 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Quantum Random Access Optimization"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The Quantum Random Access Optimization (QRAO) module is designed to enable users to leverage a new quantum method for combinatorial optimization problems [1]. This approach incorporates Quantum Random Access Codes (QRACs) as a tool to encode multiple classical binary variables into a single qubit, thereby saving quantum resources and enabling exploration of larger problem instances on a quantum computer. The encodings produce a local quantum Hamiltonian whose ground state can be approximated with standard algorithms such as VQE and QAOA, and then rounded to yield approximation solutions of the original problem.\n",
+    "\n",
+    "QRAO through a series of 3 classes:\n",
+    "1. The encoding class (`QuantumRandomAccessEncoding`): This class encodes the original problem into a relaxed problem that requires fewer resources to solve.\n",
+    "2. The rounding schemes (`SemideterministicRounding` and `MagicRounding`): This scheme is used to round the solution obtained from the relaxed problem back to a solution of the original problem.\n",
+    "3. The optimizer class (`QuantumRandomAccessOptimizer`): This class performs the high-level optimization algorithm, utilizing the capabilities of the encoding class and the rounding scheme.\n",
+    "\n",
+    "\n",
+    "### References\n",
+    "\n",
+    "[1] Bryce Fuller, Charles Hadfield, Jennifer R. Glick, Takashi Imamichi, Toshinari Itoko, Richard J. Thompson, Yang Jiao, Marna M. Kagele, Adriana W. Blom-Schieber, Rudy Raymond, Antonio Mezzacapo, *Approximate Solutions of Combinatorial Problems via Quantum Relaxations,*  [arXiv:2111.03167](https://arxiv.org/abs/2111.03167)"
```

Comment:
You don't have to enumerate all authors. You can shorten it like `Bryce Fuller et al.`

Reply:
```suggestion
    "[1] Bryce Fuller et al., *Approximate Solutions of Combinatorial Problems via Quantum Relaxations,*  [arXiv:2111.03167](https://arxiv.org/abs/2111.03167)"
```